packages/esm-patient-chart-app/src/clinical-views/utils/helpers.ts

```diff
@@ -0,0 +1,192 @@
+import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+import { type FormSchema } from '@openmrs/esm-form-engine-lib';
+import { formatDate, parseDate, formatDatetime, type Concept, age } from '@openmrs/esm-framework';
+import { type Observation, type Encounter } from '../../encounter-list/types';
+import { esmPatientChartSchema } from '../../config-schema';
+import { type TFunction } from 'i18next';
+
+type LaunchAction = 'add' | 'view' | 'edit' | 'embedded-view';
+
+export function launchEncounterForm(
+  form: FormSchema,
+  action: LaunchAction = 'add',
+  onFormSave: () => void,
+  title?: string,
+  encounterUuid?: string,
+  intent: string = '*',
+  workspaceWindowSize?: 'minimized' | 'maximized',
```

Comment:
This property isn't used at all, which seems silly because it is often given a value.